src/main/java/uk/gov/digital/ho/hocs/search/client/elasticsearchclient/ElasticSearchClient.java

```diff
@@ -49,7 +41,7 @@ public ElasticSearchClient(ObjectMapper objectMapper,
     }
 
     public CaseData findById(UUID uuid) {
-        GetRequest getRequest = new GetRequest(index, uuid.toString());
+       GetRequest getRequest = new GetRequest(index, uuid.toString());
```

Comment:
Did you do this to trigger OCD?

Reply:
I think IntelliJ did this. We really need a linter!